heaps/heap_sort.py

```diff
@@ -1,8 +1,21 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n log n)
+        Space Complexity: O(1)
     """
```

Comment:
👍  The space complexity you have here is O(n) because you 're building a heap of `n` nodes.